packages/store/src/workspace/page.ts

```diff
@@ -270,6 +279,42 @@ export class Page extends Space<FlatBlockMap> {
     });
   }
 
+  // TODO: remove hard coded types
+  updateColumnValue(columnId: string, newType: 'select' | 'rich-text') {
```

Comment:
@zqran this is a generic workaround, we will further improve this

Reply:
👌